packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -141,3 +141,21 @@ def test_files_api(dcos_api_session):
                 app_task['slaveId'], marathon_framework_id, app_task['id'], required_sandbox_file)
 
             assert content, 'File {} should not be empty'.format(required_sandbox_file)
+
+
+def test_if_ucr_app_runs_in_new_pid_namespace(dcos_api_session):
+    app, test_uuid = get_test_app_in_ucr()
+
+    ps_output_file = 'ps_output'
+    ps_cmd = 'ps ax -o pid= > {};'.format(ps_output_file)
+
+    app['cmd'] = ps_cmd + app['cmd']
```

Comment:
Why the `+ app['cmd']`? Is there some reason you want the python_test_server running after your ps command?
Seems like this could be a metronome job.